### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by brytebee**
THE NEW YORK TIMES ARTICLE PAGE
In this project, I and my partner/ collaborator have learned new things together.

We have fixed the linter errors and also made the reviewer's code corrections. 

We hope this correction meets the requirements.

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far, team. 👍 

- You've done some good progress so far.

## However, there are some changes required 👀 🧰 🔨 🔧 

- Please remember to add a proper README file for this project.
- Please add a proper project description to this Pull Request!
- Please check the automatic linter tests that are failing in this Pull Request. There are a few, and you've done some good progress in this project, so I'll do a full code review instead of invalidating your project just yet. But please, remember to fix these linter errors!
- The New York Times logo at the top of the page should be an embedded image, not just text.
- The buttons at the right side of the top bar are too narrow on the sides. Please make them a bit wider to look more like the originals.
image
- The images in your central section contain overflowing text. Please fix it:

image

- Your top navbar should have a fixed position and stick to the top of the screen, like this (taken from the original page):
image

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @brytebee  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this project can be approved.

## REQUIRED

* [x] Please, make all the changes as requested by the previous TSE.
* [ ] In the navbar, you should have the NY Times image logo embedded in the centre (link to page) (screenshot from Pathwright)
* [ ] The NY Times logo in the navbar should be a link (link to page) (screenshot from Pathwright)
* [ ] Add some margin/padding to the `SUBSCRIBE NOW` and `LOGIN` buttons in the navbar.
* [ ] Your navbar should have a fixed position (it sticks to the top part of the screen) (link to page) (screenshot from Pathwright)
* [ ] You're expected to align elements with a combination of all three techniques (float, flex and grid). (Screenshot from Pathwright) and (Screenshot from Odin)

### Make the requested changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+  line-height: 1.1rem;
+  padding: 10px 0;
+  font-size: 0.7rem;
+  opacity: 0.6;
+}
+
+.last-caption-links:hover {
+  text-decoration: none;
+  color: blue;
+}
```
Comment (andresporras3423) [inline styles.css]: Group this selector with the selector on line 242

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+  padding: 0 7%;
+}
+
+.copyright-link {
+  text-decoration: none;
+}
+
+.copyright-link:hover {
+  text-decoration: underline;
+}
```
Comment (andresporras3423) [inline styles.css]: Group this selector with the selector on line 253

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+.copyright {
+  display: flex;
+  font-size: 0.8em;
+  justify-content: space-between;
+  padding: 0 7%;
+}
+
+.copyright-link {
+  text-decoration: none;
+}
```
Comment (andresporras3423) [inline styles.css]: Group this selector with the selectors on lines 331 and 116.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Bright Atsighi** 
Good job with the improvements for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [x] The NY Times logo, in the navbar, should be a link.
- [x] Your project should align elements with a combination of all three techniques (float, flex and grid).
- [x] In your CSS file, group selectors with the same style definition.
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🔧
### Hi There ✋🏽
 Nice work so far team! However, you haven't made any of the changes required by the previous Code Reviewer. Please do and then submit for another code review.
Feel free to reach out to me on slack if you have any questions, Witah Georjane 🤝

Good luck on your project and Happy Coding 💻

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team  ,  you have done a brilliant job with this project. :clap:
#### You can merge !!!

**Pull request merged**